### PR TITLE
Image Block: Show image blocks with external URLs and support resizing

### PR DESCRIPTION
**Pull request by youknowriad**
related #5812 

This PRs fixes the image block when using an external URL (the images were not showing) and also adds resizing support using the inspector controls to these images.

**Testing instructions**

 - Check that the images in the demo content show up properly
 - Check that you can resize them using the inspector controls
 - Check that resizing using the inspector controls also work for media library inserted images.

### CODE REVIEW

**Comment by jasmussen** [comment]
Nice! works for me 👍 👍

**Comment by noisysocks** [comment]
👍 code looks good. Thanks for fixing that up, Riad!

File: blocks/library/image/block.js [hunk 1]
```diff
@@ -221,84 +219,85 @@ class ImageBlock extends Component {
 
 		const isResizable = [ 'wide', 'full' ].indexOf( align ) === -1 && ( ! viewPort.isExtraSmall() );
 
+		const getInspectorControls = ( imageWidth, imageHeight )=> (
```
Comment (noisysocks) [inline blocks/library/image/block.js]: Huh, weird that the linter didn't complain about the missing space here.

Comment (noisysocks) [reply blocks/library/image/block.js]: Fixed in https://github.com/WordPress/gutenberg/pull/6263.

**Pull request merged**